ember-cli/2017-06/june-29.md

```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-06-29
+
+## Attendees
+
+
+## Agenda

```

Comment:
https://github.com/ember-cli/rfcs/pull/105 Use optionalDependencies for addon resolution.